Model/Datasource/Oracle.php

```diff
@@ -443,7 +446,8 @@ function _execute($sql, $params = array(), $prepareOptions = array()) {
 /**
  * Fetch result row
  *
- * @return array
+ * @param string $sql optional. Some SQL to be executed
+ * @return the fetched row as an array
```

Comment:
You've lost the declaration of the return type as array.
`@return array The fetched row`